src/main/java/manufacturing/dao/jdbc/ManufacturerDaoJdbc.java

```diff
@@ -0,0 +1,119 @@
+package manufacturing.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import manufacturing.dao.ManufacturerDao;
+import manufacturing.exception.DataProcessingException;
+import manufacturing.injections.Dao;
+import manufacturing.model.Manufacturer;
+import manufacturing.util.ConnectionController;
+
+@Dao
+public class ManufacturerDaoJdbc implements ManufacturerDao {
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        try (Connection connection = ConnectionController.getConnection()) {
+            String query = "INSERT INTO manufacturers (name, country) VALUES (?, ?)";
+            PreparedStatement insert =
+                    connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS);
+            insert.setString(1, manufacturer.getName());
+            insert.setString(2, manufacturer.getCountry());
+            insert.executeUpdate();
+            ResultSet resultSet = insert.getGeneratedKeys();
+            if (resultSet.next()) {
+                manufacturer.setId(resultSet.getObject(1, Long.class));
+            }
+            insert.close();
+            resultSet.close();
```

Comment:
it is optional to close ResultSet before Statement, because when a Statement object is closed, its current ResultSet object, if one exists, is also closed.
https://docs.oracle.com/en/java/javase/11/docs/api/java.sql/java/sql/Statement.html#close()